ClamavPlugin.inc.php

```diff
@@ -329,11 +303,7 @@ function _clamDaemonShortPolling($socket, $delay = 100000000, $intervals = 10) {
 					return Array('safe' => false, 'message' => substr($output, 11));
 				}
 			}
-		}
-		if($failover === "block") {
-			return Array('safe' => false, 'message' => 'timeout');
-		} else {
-			return Array('safe' => true, '');
+			throw new ClamScanFailureException("ClamAV failed to scan the file");
```

Comment:
When adding a thrown exception it is best practice to document the expected exception with a `@throws` statement in the phpdocblock.